xml/System.Web.Security.AntiXss/AntiXssEncoder.xml

```diff
@@ -142,32 +142,32 @@
 |Character(s)|Description|  
 |--------------------|-----------------|  
 |A-Z|Uppercase Latin alphabetic characters|  
-||a-z|Lowercase Latin alphabetic characters|  
+|a-z|Lowercase Latin alphabetic characters|  
 |0-9|Numbers|  
-|!|Exclamation mark|  
-|#|Number sign, hash|  
-|$|Dollar sign|  
-|%|Percent sign|  
+|\!|Exclamation mark|  
+|\#|Number sign, hash|  
+|\$|Dollar sign|  
+|\%|Percent sign|  
 |( )|Parentheses|  
-|*|Asterisk|  
-|+|Plus sign|  
-|,|Comma|  
-|-|Hyphen, minus|  
-|.|Period, dot, full stop|  
-|/|Slash|  
-|:|Colon|  
-|;|Semicolon|  
-|=|Equals sign|  
-|?|Question mark|  
-|@|Commercial at-sign|  
-|[ ]|Square brackets|  
-|&#92;|Backslash|  
-|^|Caret|  
-|_|Underscore|  
+|\*|Asterisk|  
+|\+|Plus sign|  
+|\,|Comma|  
+|\-|Hyphen, minus|  
+|\.|Period, dot, full stop|  
+|\/|Slash|  
+|\:|Colon|  
+|\;|Semicolon|  
+|\=|Equals sign|  
+|\?|Question mark|  
+|\@|Commercial at-sign|  
+|\[ \]|Square brackets|  
+|\\|Backslash|  
```

Comment:
This has rendered incorrectly:
![image](https://user-images.githubusercontent.com/10886961/59058614-e5c2fb00-8851-11e9-83b7-5661e987a644.png)
I'm not sure why. Possibly add a space after the escaped backslash?